sharing_portal/views.py

```diff
@@ -616,6 +616,10 @@ def artifact(request, artifact, version_slug=None):
             messages.error(request, message)
 
     git_content = [method for method in access_methods if method["protocol"] == "git"]
+    feedback_url = ""
+    if len(git_content) > 0:
+        # git repository is /<repository>.git format
+        feedback_url = git_content[0]["remote"].replace(".git", "/issues")
```

Comment:
This should explicitly check for github, and may need to extract the username/repo name to be correct, as the url might be `git://` too. For trovi, we use a library to do so: https://github.com/ChameleonCloud/trovi/blob/main/trovi/storage/backends/git.py#L57